### PR TITLE
[MRG] warning about too small n_permutations

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by dengemann** [comment]
This is a demo PR from our MNE workshop.

File: tutorials/plot_stats_cluster_1samp_test_time_frequency.py [hunk 1]
```diff
@@ -84,8 +84,9 @@
 # Compute statistic
 # -----------------
 threshold = 2.5
+n_permutations = 100  # Warning: 100 is too small please use a bigger number
```
Comment (dengemann) [inline tutorials/plot_stats_cluster_1samp_test_time_frequency.py]: Improve sentence.

Something like:

# Warning: 100 is too small for real-world analysis.

File: tutorials/plot_stats_cluster_1samp_test_time_frequency.py [hunk 2]
```diff
@@ -84,8 +84,9 @@
 # Compute statistic
 # -----------------
 threshold = 2.5
+n_permutations = 100  # Warning: 100 is too small for real-world analysis.
```
Comment (larsoner) [inline tutorials/plot_stats_cluster_1samp_test_time_frequency.py]: Good enough for me. If you want to get fancy you can put this in the `###` block above with a `.. warning:: ` so it gets a special angry-looking box around it

**Comment by agramfort** [comment]
let's say it's good enough :)

**Comment by dengemann** [comment]
Now we only need to fix that in all the other examples :)

**Pull request merged**